### DISCUSSION TITLE
다양한 Header 종류를 어떻게 만들 것인가에 대한 논의

### DISCUSSION THREAD

**Discussion by hy57in** [teamsindy20/sobok#45] [Design]
# Header 디자인 종류
### 1. 홈
![image](https://user-images.githubusercontent.com/60775453/131302284-31ca5885-39be-4c75-8eb5-2d6e66ce30a2.png)
### 2. 주소지 설정
![image](https://user-images.githubusercontent.com/60775453/131302339-e8b3f4bd-36a9-4e2b-bec6-1b7728f6fdcd.png)
### 3. 지도
![image](https://user-images.githubusercontent.com/60775453/131302380-1277e0af-c75c-4f7f-bf4d-19929e772159.png)
### 4. 검색
![image](https://user-images.githubusercontent.com/60775453/131302440-d3f1e6c5-8a8c-4855-a4c5-d741ea29de6c.png)
### 5. 검색 결과
![image](https://user-images.githubusercontent.com/60775453/131302481-6d188bd5-7afb-4553-8d24-18ba76576a6d.png)
### 6. 매장상세
![image](https://user-images.githubusercontent.com/60775453/131302564-d7368524-b2d4-439b-9a46-c88d00f666e4.png)
### 7. 매장소식
![image](https://user-images.githubusercontent.com/60775453/131302606-029fcc71-22a1-48e8-a31b-8cee316f5831.png)

# 논의할 사항
- 각 Header에서 fixed 영역은?
- component를 어떻게 분리시킬 것인지?
- Layout에 담을 것인지, Page에 담을 것인지
- tab같은 경우는 현재 antd를 사용하는데 tab과 같이 있는 header의 경우 fixed를 어떻게 할 것인지?

**Comment by gwak2837** [comment]
### Layout vs Page

Layout 컴포넌트는 각 Page 별 공통되는 HTML이 있을 때 사용하고, 페이지엔 그 페이지만의 HTML을 넣는 것을 권장해요. 왜냐하면 Layout을 사용하면 url이 변경돼도 Layout 컴포넌트 상태가 유지되기 때문이에요.

### Fixed header

`position: fixed`는 CSS를 잘 적용하면 될 거 같아요..!!
